.travis.yml

```diff
@@ -1,183 +1,190 @@
+            - sourceline: 'ppa:beineri/opt-qt532-trusty'
+            # g++ & gcc >= 4.9
+            - ubuntu-toolchain-r-test
+          packages:
+            - git
+            - build-essential
+
+            #### Replacement of: qt5-qmake, qt5-default, qttools5-dev-tools
+            - qt53base
+            - qt53tools
+
+            #### Using g++ >= 4.9
+            - g++-4.9
+            - gcc-4.9
+
+      before_install:
+        ##### Clean cache folder
+        - rm -rf $BUILD_DST_PATH/*
+
+      before_script: &build_before_script
+        #### Use qt5 == 5.3 instead other version
+        - source /opt/qt53/bin/qt53-env.sh && qmake --version
+
+        ##### Use gcc y cpp == 4.9 instead other version
+        - export CC=gcc-4.9 CXX=g++-4.9
+
+      script:
+        - qmake QMAKE_CXX=$CXX QMAKE_CC=$CC QMAKE_LINK=$CXX DESTDIR=$BUILD_DST_PATH
+
+        # Building flameshot
+        - make -j$(nproc)
+
+        # Running flameshot tests
+        - make check -j$(nproc)
+        - ls -alhR
+
+      after_script:
+        #### Wait preventing cache doesn't update
+        - sleep 10
+
+############## Packaging AppImage ##############
+    - stage: Packaging
+      #language: cpp
+      #compiler: gcc
+      cache:
+        - directories:
+          - *build_dst_path
+          ### > Save the appimage directory to reuse in other jobs
+          - &appimage_dst_path build-appimage
+      addons:
+        apt:
+          sources:
+            # qt5 >= 5.3
+            - sourceline: 'ppa:beineri/opt-qt532-trusty'
+          packages:
+            #### Replacement of: qt5-qmake, qt5-default, qttools5-dev-tools
+            - qt53base
+            # - qt53tools
+      before_install:
+        ##### Clean cache folder
+        - rm -rf $APPIMAGE_DST_PATH/*
+
+      before_script:
+        #### Use qt5 == 5.3 instead other version
+        - source /opt/qt53/bin/qt53-env.sh && qmake --version
+
+        ##### Download and install linuxdeployqt
+        - wget -c "https://github.com/probonopd/linuxdeployqt/releases/download/continuous/linuxdeployqt-continuous-x86_64.AppImage" -O linuxdeployqt
+        - chmod +x linuxdeployqt
+        ## Extract linuxdeployqt in 'squashfs-root/'
+        - ./linuxdeployqt --appimage-extract
+        # - ls -R | grep ":$" | sed -e 's/:$//' -e 's/[^-][^\/]*\//--/g' -e 's/^/   /' -e 's/-/|/'
+            #|-squashfs-root
+            #|---usr
+            #|-----bin
+            #|-----lib
+            #|-----share
+            #|-------doc
+            #|---------libicu52
+            #|---------libpcre3
+            #|---------qt59base
+            #|-----translations
+
+      script:
+        #### Prepare packing
+        - mkdir appdir
+        - cp $BUILD_DST_PATH/flameshot appdir/
+        - cp docs/desktopEntry/package/* appdir/
+        - cp img/flameshot.png appdir/
+        - ls -alhR appdir/
+
+        #### Configure env vars
+        - unset QTDIR; unset QT_PLUGIN_PATH ; unset LD_LIBRARY_PATH
```

Comment:
You can do `unset QTDIR QT_PLUGIN_PATH  LD_LIBRARY_PATH`

Reply:
I will modify it. Actually I don't remember why i put that, i'm going to try to omit it or comment why i execute it